packages/create-cloudflare/templates/vike/experimental-c3.ts

```diff
@@ -0,0 +1,30 @@
+import { logRaw } from "@cloudflare/cli";
+import { runFrameworkGenerator } from "frameworks/index";
+import type { TemplateConfig } from "../../src/templates";
+import type { C3Context } from "types";
+
+const generate = async (ctx: C3Context) => {
+	await runFrameworkGenerator(ctx, [
+		ctx.project.name,
+		// Note: the vike create CLI supports a `--cloudflare` argument for creating cloudflare-ready
+		//       projects, that works great but here we don't want to use that because we do want to
+		//       exercise the general autoconfig/`wrangler setup` functionality (which amongst other
+		//       things lets up ensure that we can support the migration of existing vike projects)
```

Comment:
Wouldn't it be better to use the flag in C3, and leave the autoconfig stuff for other sites created without it?

Reply:
Seems a bit icky to add a template just for e2e tests, but if there's no better way then sure